### DISCUSSION TITLE
I can't find the definition of certain classes

### DISCUSSION THREAD

**Discussion by kb1995** [saadeghi/daisyui#699] [Q&A] [answered]
I might be looking at the completely wrong place, but I can't find the definitions for `btn-lg` variant anywhere in the codebase. Are these generated via postcss?

**Comment by kb1995** [comment, marked as answer]
Found it!

https://github.com/saadeghi/daisyui/blob/master/src/utilities/unstyled/button.css

**Reply by saadeghi** [reply]
Yup. All responsive modifier classes are in `/src/utilities`

**Reply by kb1995** [reply]
@saadeghi Is it correct calling `btn-lg` a responsive modifier? In many cases, you'd like to use btn-lg or btn-sm in the same viewport as normal btn

**Reply by saadeghi** [reply]
It's a "modifier" class because it modifies the default `btn` style.

And it's not necessarily responsive but it *can be* responsive (with prefixes like `lg:btn-lg`)
